code/modules/mob/living/carbon/examine.dm

```diff
@@ -157,24 +157,28 @@
 //SKYRAT EDIT REMOVAL - MOVED - MEDICAL
 /*
 /mob/living/carbon/examine_more(mob/user)
-	if(!all_scars)
-		return ..()
+	. = ..()
+	. += span_notice("<i>You examine [src] closer, and note the following...</i>")
+
+	//On closer inspection, this man isnt a man at all!
+	var/list/covered_zones = get_covered_body_zones()
+	for(var/obj/item/bodypart/part as anything in bodyparts)
+		if(part.body_zone in covered_zones)
+			continue
+		if(part.limb_id != (dna.species.examine_limb_id ? dna.species.examine_limb_id : dna.species.id))
+			. += "[span_info("[p_they(TRUE)] [p_have()] \an [part.name].")]"
```

Comment:
Did you bring that to the modular version?

Reply:
I did now!